.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2007-present the original author or authors.
```

Comment:
I'm not sure "-present" is valid but IANAL...

Reply:
This file is maintained by takari, and is what `mvn -N io.takari:maven:0.7.7:wrapper` created. When moving to the maven v3.7.0 version I believe it had more formal apache maven headers and copyright.